util/agentstore.go

```diff
@@ -27,9 +34,11 @@ func AgentStore() *AgentStorage {
 
 func NewAgentStore(filename string) error {
 	agentDefault := map[string]string{
-		"id":      "",
-		"address": "",
-		"tx":      "",
+		"id":                "",
+		"address":           "",
+		"tx":                "",
+		string(OwnerKey):    "",
+		string(OperatorKey): "",
```

Comment:
We should use the `AgentOwner` and `AgentOperator` queries from the sdk to get these. This way we don't accidentally pass a stale address somewhere from a failed rewrite or a change of owner made from another place

Reply:
I'm not sure I follow ... I think this is just initializing blank fields in the agent.toml file before the agent has even been created.